ci/terraform/volumes.tf

```diff
@@ -0,0 +1,6 @@
+resource "harvester_volume" "vol001" {
```

Comment:
It seems not be attached

Reply:
Thanks for the reminder!
I update the VM.tf. Please help to recheck. Thanks!